server.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const express = require('express');
+
+// Constants
+const PORT = 8080;
```

Comment:
Perhaps it's my ignorance of docker but why expose 3000 if this is using 8080? Also, maybe this should be an environment variable so it can be changed without changing code?